### PR TITLE
Contribution Guide

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: samhita-alla <aallasamhita@gmail.com>

### CODE REVIEW

File: rsts/community/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+*A piece of good news -- You can be added as a committer to any ``flyteorg`` repo as you become more involved with the project.*
+
+Example PR for your reference: `GitHub PR <https://github.com/flyteorg/flytepropeller/pull/242>`_. A couple of checks are introduced to help in maintaining the robustness of the project. 
+
+#. To get through DCO, sign off on every commit. (`Reference <https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md>`_) 
+#. To improve code coverage, write unit tests to test your code.
+
+.. note::
+    Format your Go code using the ``go fmt`` command and Python code with ``black`` (using ``make fmt`` command which contains both black and isort).
```
Comment (kumare3) [inline rsts/community/contribute.rst:29]: we actually use `go imports`.
We run `golang-ci` for linting and  it follows goimports formating
https://github.com/flyteorg/flytepropeller/blob/master/boilerplate/lyft/golang_test_targets/Makefile#L11-L13

File: rsts/community/contribute.rst [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+#. To get through DCO, sign off on every commit. (`Reference <https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md>`_) 
+#. To improve code coverage, write unit tests to test your code.
+
+.. note::
+    Format your Go code using the ``go fmt`` command and Python code with ``black`` (using ``make fmt`` command which contains both black and isort). 
+
+Environment Setup
+*****************
+
+.. figure:: ../images/dependency_graph.png
```
Comment (kumare3) [inline rsts/community/contribute.rst:34]: so for all the images we are using this deatched branch - 
https://github.com/flyteorg/flyte/tree/static-resources
Can you move the image to there and reference it from here. E.g check the flowchart here - https://github.com/flyteorg/flyte/blob/master/rsts/plugins/extend/intro.rst or the gif's etc

This is to keep the size of the git clone to be small. You will need to clone and push it as a detached head branch

**Comment by kumare3** [comment]
this looks great, one comment then +1

**Comment by samhita-alla** [comment]
Added the suggested changes.

**Review by kumare3 — APPROVED** [review]

**Pull request merged**